### PR TITLE
CY-3301 Separate node services per node type

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

File: rest-service/manager_rest/cluster_status_manager.py [hunk 1]
```diff
@@ -100,6 +100,36 @@
...
 }
+SERVICE_ASSIGNMENTS = {
+    CloudifyNodeType.DB: [
+        'etcd',
+        'haproxy',
+        'nginx',
+        'node_exporter',
+        'patroni',
+        'postgres_exporter',
+        'postgresql-9.5',
```
Comment (geokala) [inline rest-service/manager_rest/cluster_status_manager.py:111]: This service shouldn't be on a DB node, only on an AIO node.

Comment (geokala) [reply rest-service/manager_rest/cluster_status_manager.py:111]: (though we do have other code that handles that, so not necessarily a major concern at this point)

**Review by geokala — APPROVED** [review]
Approved- I still feel like this part of the code is overcomplicated but I think cleaning it up is not a task to be done close to release.

File: rest-service/manager_rest/cluster_status_manager.py (hunk 1 above)
Comment (mateumann) [reply rest-service/manager_rest/cluster_status_manager.py:111]: yes, exactly my attitude here

**Comment by mateumann** [comment]
Yes it is too complicated, as is the returned data structure.  But (this comment concerns the latter) we should maintain backward compatibility.

**Comment by geokala** [comment]
Agreed, back compat is important, but let's review the actual code structure at some point and see if we can make it easier to follow... somehow. <smiles>

**Pull request merged**